src/screens/DomainScreen/elements/Header/index.js

```diff
@@ -94,7 +94,7 @@ const styles = StyleSheet.create({
   },
   containerFollowers: {flexDirection: 'row'},
   followers: {
-    color: COLORS.anon_primary,
+    color: COLORS.signed_primary,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing the color of the text from `COLORS.anon_primary` to `COLORS.signed_primary` is a minor change and should not affect the functionality of the code. However, make sure that the new color provides sufficient contrast for readability and aligns with the overall color scheme of your application.
```diff
-    color: COLORS.anon_primary,
+    color: COLORS.signed_primary,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->